plugins/GSdx/GSRendererHW.cpp

```diff
@@ -796,7 +796,7 @@ void GSRendererHW::Hacks::SetGameCRC(const CRC::Game& game)
 		m_oi = &GSRendererHW::OI_PointListPalette;
 	}
 
-	bool hack = theApp.GetConfig("UserHacks_ColorDepthClearOverlap", 0) && theApp.GetConfig("UserHacks", 0);
+	bool hack = theApp.GetConfigB("UserHacks_ColorDepthClearOverlap") && theApp.GetConfigB("UserHacks");
 	if (hack && !m_oi) {
```

Comment:
you seem to be calling `` theApp.GetConfigB("UserHacks")`` quite a few time on this file, maybe cache it as a member variable ?